src/gsi/read_satwnd.f90

```diff
@@ -244,7 +247,7 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
   equivalence(r_sprvstg(1,1),c_sprvstg)
   equivalence(rstation_id,c_station_id)
 
-  data hdrtr_v1 /'SAID CLAT CLON YEAR MNTH DAYS HOUR MINU SWCM SAZA OGCE SCCF SWQM'/ ! OGCE replaces GCLONG, OGCE exists in old and new BUFR
+  data hdrtr_v1 /'SAID CLAT  CLON  YEAR MNTH DAYS HOUR MINU SWCM SAZA OGCE SCCF SWQM'/ ! OGCE replaces GCLONG, OGCE exists in old and new BUFR
```

Comment:
Is there a reason why the extra space around CLON is added here?

Reply:
The extra space is explained by the line after that one:
![image](https://github.com/user-attachments/assets/fc451934-ad0e-4dd5-aebb-80cfe5728381)